.vsts-ci/ci.yml

```diff
@@ -22,17 +22,20 @@ stages:
   - template: ./templates/releaseBuildPhase.yml
     parameters:
       channel: lts
+      vmImage: ubuntu-18.04
```

Comment:
why would this not be `PSMMSUbuntu18.04-Secure`? Is it because it just gets set to `PSMMSUbuntu18.04-Secure` as a default in releaseBuildPhase.yml?

Reply:
use public images for testing